tables/topic_combinations.md

```diff
@@ -1 +1,56 @@
-topic_combinations.md
+# tables/topic_combinations
+
+
+## What is topic_combinations?
+The `topic_combinations` table lists out the available `topic` combinations for each possible set of `geography_combinations`.
```

Comment:
I know this is also kind of waiting on a glossary/terminology page, might be worth linking topic to `tables/topics` and `tables/geography_combinations` just to provide context and explain a little more.

Reply:
sure, added

Reply:
Thanks - Look to still be missing a link to /topics?

Reply:
had a chat in slack, and think we resolved this